app.js

```diff
@@ -13,21 +13,40 @@ app.use(morgan('dev'));
 
 app.use(bodyParser.json());
 
+app.use(function(req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept, Authorization");
+    res.header('Access-Control-Allow-Methods', 'PUT, POST, GET, DELETE, OPTIONS');
+    next();
+});
 
-
-mongoose.connect('mongodb+srv://Tuyizere:anastasetuyizere@cluster0.6kcyh.mongodb.net/NewBrand?retryWrites=true&w=majority',
+mongoose.connect('mongodb+srv://AnastaseTuyizere:anastasetuyizere@cluster0.so3mc.mongodb.net/MyBrand?retryWrites=true&w=majority',
```

Comment:
Consider using `.env` so you don't expose your credentials (email and password) to the public when you push to github.